packages/@aws-cdk/aws-stepfunctions/test/test.states-language.ts

```diff
@@ -729,5 +729,5 @@ class FakeResource implements stepfunctions.IStepFunctionsTaskResource {
 }
 
 function render(sm: stepfunctions.IChainable) {
```

Comment:
It is not.

Reply:
Since all implementers of `IChainable` are constructs, it should be possible to simply have `IChainable` extend `IConstruct`.

Reply:
A `Chain` is an `IChainable` and yet isn't a construct.

Reply:
Sorry... Didn't realize it's a unit test :-) forget about it.

[although it should be straightforward to implement `IConstruct` in `Chain` because it always has a `startState`, so you could implement `node` via means of `this.startState.node` and suddenly `IChainable` _is_ an `IConstruct`.... Just saying...